common/config/rush/common-versions.json

```diff
@@ -1,8 +1,50 @@
+/**
+ * This configuration file specifies NPM dependency version selections that affect all projects
+ * in a Rush repo.  For full documentation, please see https://rushjs.io
+ */
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/common-versions.schema.json",
+
+  /**
+   * A table that specifies a "preferred version" for a dependency package. The "preferred version"
+   * is typically used to hold an indirect dependency back to a specific version, however generally
+   * it can be any SemVer range specifier (e.g. "~1.2.3"), and it will narrow any (compatible)
+   * SemVer range specifier.  See the Rush documentation for details about this feature.
+   */
   "preferredVersions": {
-    "webpack": "3.11.0",
```

Comment:
Not sure why `webpack@3.11.0` was listed as preferred here, since we've been on 4.x for awhile. This turned out to be the main cause of the error finding `webpack/schemas/WebpackOptions.json`: `common/temp/node_modules/webpack` referred to version 3.11.0, and in Webpack 3 the config file has a different name.

Reply:
`@storybook/react@3.4.11` still uses webpack 3. I don't want to take on upgrading to Storybook 4 in this PR since it's already rather large, but I could do that later if you want. (found an [upgrade guide](https://medium.com/storybookjs/migrating-to-storybook-4-c65b19a03d2c) but I haven't done enough with Storybook to have any sense of how hard the upgrade would be)